flink-core/src/main/java/org/apache/flink/api/connector/sink2/Committer.java

```diff
@@ -38,7 +38,7 @@
 @Public
 public interface Committer<CommT> extends AutoCloseable {
     /**
-     * Commit the given list of {@link CommT}.
+     * Commit the given list of {@link CommitRequest}.
```

Comment:
`CommT` is a generic parameter and we can't visit the link.

Reply:
nit: If you are changing this line, you could change List to <code>Collection</code> and all the references of list to collection.